### ISSUE TITLE
Image not accessible

### ISSUE THREAD

**Issue by calumabarnett** [ministryofjustice/data-and-analytics-engineering#38]
This image and possibly others are not accessible and don't contain sufficient alt text. Are there more accessible ways that we could present this information without using an image? Using other components?

https://github.com/ministryofjustice/data-and-analytics-engineering/blob/317e763abb67c03b91cc3f4eff35f3e24b31944d/src/content/about/index.md?plain=1#L16

**Comment by SoumayaMauthoorMOJ** [comment]
@calumabarnett how about just proving more alt text? Not sure what you mean by components?

**Comment by calumabarnett** [comment]
@SoumayaMauthoorMOJ that would be fine I think. By other components I was thinking could some images be displayed using HTML elements making them directly readable and navigable by screen readers, etc.

**Comment by SoumayaMauthoorMOJ** [comment]
Ok I'll update the alt text in all the images, and will raise a ticket about investigating HTML elements
